tests/conftest.py

```diff
@@ -9,10 +9,10 @@
 from drf_integrations import integrations
 from drf_integrations.integrations.base import BaseIntegration
 from tests.integration_samples import (
-    TestInternalIntegration,
-    TestInternalWithFormIntegration,
-    TestLocalIntegration,
-    TestLocalWithFormIntegration,
+    InternalIntegrationTest,
```

Comment:
having test at the start of the name confuses pytest as these are not true pytest classes